dependency-check-suppressions.xml

```diff
@@ -76,4 +76,8 @@
         <notes><![CDATA[jackson:2.9.9.jar]]></notes>
         <cve>CVE-2020-8840</cve>
     </suppress>
+    <suppress>
```

Comment:
Please don't add that in suppression, we need to fix this, for that please check spring boot release version, if you upgrade to latest version which is 2.2.5 , this vulnerabilities will be gone, but we need to make application specific changes.